seed/publish.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
I would prefer not to have scripts in the middle of the resources. Can we put this script in the root or in a folder just for scripts?

Reply:
I agree, but I would do it as part of a different PR in which we integrate the publish with Travis